src/main/java/teammates/logic/core/CoursesLogic.java

```diff
@@ -115,19 +115,39 @@ public void createCourseAndInstructor(String instructorGoogleId, String courseId
         }
     }
 
+    /**
+     * Get the {@link CourseAttributes} using the courseId.
```

Comment:
I skipped the 'a'. Since it returns `CourseAttributes `for the given `courseId`. Tell me if you disagree :)